packages/plugins/model-evaluate/object-detection-detectron-model-evaluate/src/index.ts

```diff
@@ -22,13 +18,12 @@ const detectronModelEvaluate: ModelEvaluateType = async (data: CocoDataset, mode
     cfg.DATASETS.TEST = [ "test_dataset" ];
 
     const evaluator = COCOEvaluator("test_dataset", cfg, false, boa.kwargs({ output_dir: modelDir }));
-      
     const val_loader = build_detection_test_loader(cfg, "val_dataset");
-
-    const val_result = inference_on_dataset(trainer.model, val_loader, evaluator);
+    const result = inference_on_dataset(trainer.model, val_loader, evaluator);
```

Comment:
inference_on_dataset method should return a dict which contains AP50, AP75 for each class and MAP (depends on config). So I think it's not valid when use `result > expectResult`

Reply:
Thanks @utkobe, But I can't find any about the keys list at https://detectron2.readthedocs.io/_modules/detectron2/evaluation/evaluator.html#DatasetEvaluator.evaluate, is there the kind of list?

Reply:
could have a look at https://detectron2.readthedocs.io/_modules/detectron2/evaluation/coco_evaluation.html#COCOEvaluator.__init__    essentially it will use pycocotools to evaluate standard metrics required by coco